lib/tasks/stimulus_tasks.rake

```diff
@@ -7,6 +8,14 @@ module Stimulus
       system RbConfig.ruby, "./bin/rails", "app:template", "LOCATION=#{File.expand_path("../install/#{path}.rb",  __dir__)}"
     end
 
+    def controllers_path
+      Rails.root.join(StimulusGenerator.new(["stimulus"], generator_options, {}).options.js_root, "controllers")
+    end
+
+    def generator_options
```

Comment:
Since this isn't called anywhere else, I'd just inline it in `controllers_path`.

Reply:
I like:

```ruby
def controllers_path
  generator = StimulusGenerator.new(["stimulus"], Rails.application.config.generators.options.dig(:stimulus), {}) 
  Rails.root.join generator.options.js_root, "controllers"
end
```